test/sharness/t0060-daemon.sh

```diff
@@ -43,15 +44,15 @@ test_expect_success "ipfs peer id looks good" '
 #
 test_expect_failure "ipfs daemon output looks good" '
   STARTFILE="ipfs cat /ipfs/$HASH_WELCOME_DOCS/readme" &&
-  echo "Initializing daemon..." >expected &&
-  echo "initializing ipfs node at $IPFS_PATH" >>expected &&
-  echo "generating 4096-bit RSA keypair...done" >>expected &&
-  echo "peer identity: $PEERID" >>expected &&
-  echo "to get started, enter:" >>expected &&
-  printf "\\n\\t$STARTFILE\\n\\n" >>expected &&
-  echo "API server listening on /ip4/127.0.0.1/tcp/5001" >>expected &&
-  echo "Gateway (readonly) server listening on /ip4/127.0.0.1/tcp/8080" >>expected &&
-  test_cmp_repeat_10_sec expected actual_daemon
+  echo "Initializing daemon..." >expected_daemon &&
+  echo "initializing ipfs node at $IPFS_PATH" >>expected_daemon &&
+  echo "generating 2048-bit RSA keypair...done" >>expected_daemon &&
```

Comment:
Not sure what you mean, It passes now.

I wanted to see if it does so in the test runners before I change the expectation.